Watchman.Discord/Areas/Initialization/Services/ResponsesInitService.cs

```diff
@@ -22,37 +22,46 @@ public ResponsesInitService(ICommandBus commandBus, ResponsesGetterService respo
 
         public async Task InitNewResponsesFromResources()
         {
-            var defaultResponses = this._responsesGetterService.GetResponsesFromResources();
+            var defaultResponses = this._responsesGetterService.GetResponsesFromResources().ToList();
+            var responsesInBase = this._responsesGetterService.GetResponsesFromBase();
+            var responsesToUpdate = new List<Response>();
+            var responsesToRemove = new List<Response>();
             
-            var responsesToUpdate = this._responsesGetterService.GetResponsesFromBase().Select(baseResponse =>
+            foreach (var baseResponse in responsesInBase)
             {
-                // Update every response in the DB with availableVariables from defaultResponses
-                var matchingDefaultResponse = defaultResponses
-                    .FirstOrDefault(defaultResponse => defaultResponse.OnEvent == baseResponse.OnEvent);
+                // Check if there's a matching default response in resources
+                var matchingDefaultResponse = defaultResponses.FirstOrDefault(defaultResponse => defaultResponse.OnEvent == baseResponse.OnEvent);
                 if (matchingDefaultResponse != null)
                 {
+                    // there's a matching default response, update existing db response
                     baseResponse.UpdateAvailableVariables(matchingDefaultResponse.AvailableVariables);
                     baseResponse.SetMessage(matchingDefaultResponse.Message);
+                    responsesToUpdate.Add(baseResponse);
                 }
-                return baseResponse;
-            }).ToArray();
+                else
+                {
+                    // there's no matching default response, remove the db one
+                    responsesToRemove.Add(baseResponse);
+                }
+            }
```

Comment:
komentarze w całej tej metodzie są zbędne - kod mówi sam za siebie i jasno to widać